### PR TITLE
heal-preprod upgrade services to 2021.10

### PR DESCRIPTION
**Pull request by haraprasadj**
Link to Jira ticket if there is one:

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# preprod.healdata.org/manifest.json
## Deployment changes
- indexd
  - Include example secrets to load PostgreSQL settings for `indexd` (uc-cdis/indexd#317)

File: preprod.healdata.org/manifest.json [hunk 1]
```diff
@@ -8,26 +8,26 @@
...
+    "dashboard": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-statics:2021.10",
+    "fence": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/fence:5.3.2",
     "fluentd": "fluent/fluentd-kubernetes-daemonset:v1.2-debian-cloudwatch",
-    "hatchery": "quay.io/cdis/hatchery:feat_ecs",
-    "indexd": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/indexd:2021.09",
-    "manifestservice": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/manifestservice:2021.09",
+    "hatchery": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/hatchery:0.2.0",
+    "indexd": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/indexd:2021.10",
+    "manifestservice": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/manifestservice:2021.10",
     "metadata": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/metadata-service:1.6.1",
```
Comment (mfshao) [inline preprod.healdata.org/manifest.json:20]: I think we can also put MDS on mothly now right? 🤔

**Review by mfshao — APPROVED** [review]

File: preprod.healdata.org/manifest.json (hunk 1 above)
Comment (haraprasadj) [reply preprod.healdata.org/manifest.json:20]: 1.6.1 seems to have some additional commits compared to 2021.10. But we can use the monthly version once we go to 2021.11

Comment (haraprasadj) [reply preprod.healdata.org/manifest.json:20]: > I think we can also put MDS on mothly now right? 🤔

1.6.1 seems to have some additional commits compared to 2021.10. But we can use the monthly version once we go to 2021.11

**Pull request merged**